tests/tt_metal/tt_metal/perf_microbenchmark/1_compute_mm/test_compute_mm.cpp

```diff
@@ -158,12 +269,24 @@ int main(int argc, char** argv) {
         uint32_t M;
         uint32_t N;
         uint32_t K;
+        uint32_t dtype = 0; // bfp8
+        uint32_t fidel = 0; // lofi
         uint32_t num_tests = 10;
+        uint32_t num_blocks = 1;
+        bool matmul_block = 0;
+        bool packer_l1 = 0;
+        bool single_core = 0;
         bool fast_dispatch_mode = false;
         try {
             std::tie(M, input_args) = test_args::get_command_option_uint32_and_remaining_args(input_args, "--m", 11264);
             std::tie(N, input_args) = test_args::get_command_option_uint32_and_remaining_args(input_args, "--n", 3072);
             std::tie(K, input_args) = test_args::get_command_option_uint32_and_remaining_args(input_args, "--k", 768);
+            std::tie(dtype, input_args) = test_args::get_command_option_uint32_and_remaining_args(input_args, "--dtype", 0);
+            std::tie(fidel, input_args) = test_args::get_command_option_uint32_and_remaining_args(input_args, "--fidel", 0);
+            std::tie(matmul_block, input_args) = test_args::get_command_option_uint32_and_remaining_args(input_args, "--block", 0);
+            std::tie(packer_l1, input_args) = test_args::get_command_option_uint32_and_remaining_args(input_args, "--packer", 0);
+            std::tie(single_core, input_args) = test_args::get_command_option_uint32_and_remaining_args(input_args, "--one-core", 0);
+            std::tie(num_blocks, input_args) = test_args::get_command_option_uint32_and_remaining_args(input_args, "--num-blocks", 1);
```

Comment:
Current source comments on using these args at the top of the source file. https://github.com/tenstorrent-metal/tt-metal/blob/c43800ccd40284d7796f4393cf829ac8fe144e50/tests/tt_metal/tt_metal/perf_microbenchmark/1_compute_mm/test_compute_mm.cpp#L61-L69
I would appreciate it if you could add a brief explanation.